client/components/gravatar/index.jsx

```diff
@@ -64,10 +72,19 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		const avatarURL = this.props.tempImage ||
+			this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
 
 		return (
 			<img alt={ alt } className="gravatar" src={ avatarURL } width={ size } height={ size } onError={ this.onError } />
 		);
 	}
-} );
+}
+
+export default connect( ( state, ownProps ) => {
+	const userId = ownProps.user && ownProps.user.ID;
```

Comment:
That's neat, I'm using it now. Didn't use `false` as the fallback since we're expecting `undefined` anyway.